pino.js

```diff
@@ -214,7 +241,11 @@ function pinoWrite (obj, msg, num) {
     stream.lastTime = t.slice(8)
     stream.lastLogger = this // for child loggers
   }
-  stream.write(flatstr(s))
+  if (stream instanceof SonicBoom) {
```

Comment:
Faster way than instanceof? (Maybe if sonic boom adds flag to stream?)

Reply:
I cannot appreciate any difference here in not having an `instanceof` check in this block if it's a SonicBoom instance. This specific `instanceof` gets optimized away.